src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { router } from 'expo-router';
+import React from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+
+import styles from './styles';
+import WhiteTrash from '../../../../../assets/white-trash.svg';
+import X from '../../../../../assets/x.svg';
+// import CasesHeader from '../../../../Components/CasesHeader/CasesHeader'; address in future PR
+import { useSession } from '../../../../context/AuthContext';
+function LogOutConfirmation() {
+  const { signOut, session } = useSession();
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.line}>
+        <View style={styles.screenContainer}>
+          {/* <View style={styles.headerContainer}>
+            <CasesHeader />
+          </View> */}
+
+          <View style={styles.textContainer}>
+            <Text style={styles.topText}>
+              Are you sure you'd like to log out?
+            </Text>
+          </View>
+
+          <View style={styles.spaceBetween} />
```

Comment:
Ideally we should avoid using empty divs to create spacing. If we place the text and the the buttons within the same div, we could use the `rowGap` property.